tools/password_remover.py

```diff
@@ -23,6 +24,112 @@ def is_valid_json(invalid_json):
 
     return got_exception == False
 
+
+def tag_userdata_in_server_config(json_text, log_json_parsing_exceptions=True):
+    """
+    Content postprocessor that tags user data in a config ready for post-process redaction
+    """
+    try:
+
+        # lower case everything so that "databases" works as a key even if the JSON has "Databases"
+        # as a key.  seems like there has to be a better way!
+        json_text = json_text.lower()
+        valid_json = convert_to_valid_json(json_text)
+        parsed_json = json.loads(valid_json)
+
+        tag_userdata_in_server_json(parsed_json)
+
+        formatted_json_string = json.dumps(parsed_json, indent=4)
+        return formatted_json_string
+
+    except Exception as e:
+        msg = "Exception trying to tag config user data in {0}.  Exception: {1}".format(json_text, e)
+        if log_json_parsing_exceptions:
+            print(msg)
```

Comment:
Since `msg` is only used once, might be slightly clearer if it's just `print("Exeption....")`